persistence/persistence-api/src/main/java/org/cbioportal/persistence/CacheDisableSetting.java

```diff
@@ -0,0 +1,40 @@
+package org.cbioportal.persistence;
+
+import org.springframework.beans.factory.annotation.Value;
+
+public class CacheDisableSetting {
+
+    @Value("${ehcache.disable_all_caching:false}")
+    private String setting;
```

Comment:
`setting` here can be automatically parsed as a boolean if you change the declaration to `private Boolean setting;`

Reply:
I also think that the member should be renamed to something like `ehcacheDisabled`

Reply:
the member is now a simple boolean called "enabled"